src/legends/swatches.js

```diff
@@ -1,10 +1,10 @@
-import {path} from "d3";
-import {inferFontVariant} from "../axes.js";
-import {maybeAutoTickFormat} from "../axis.js";
-import {Context, create} from "../context.js";
-import {isNoneish, maybeColorChannel, maybeNumberChannel} from "../options.js";
-import {isOrdinalScale, isThresholdScale} from "../scales.js";
-import {applyInlineStyles, impliedString, maybeClassName} from "../style.js";
+import { path } from "d3";
+import { inferFontVariant } from "../axes.js";
+import { maybeAutoTickFormat } from "../axis.js";
+import { Context, create } from "../context.js";
+import { isNoneish, maybeColorChannel, maybeNumberChannel } from "../options.js";
+import { isOrdinalScale, isThresholdScale } from "../scales.js";
+import { applyInlineStyles, impliedString, maybeClassName } from "../style.js";
```

Comment:
```suggestion
import {path} from "d3";
import {inferFontVariant} from "../axes.js";
import {maybeAutoTickFormat} from "../axis.js";
import {Context, create} from "../context.js";
import {isNoneish, maybeColorChannel, maybeNumberChannel} from "../options.js";
import {isOrdinalScale, isThresholdScale} from "../scales.js";
import {applyInlineStyles, impliedString, maybeClassName} from "../style.js";
```

please let's stick to the prettier preferences of the project

Reply:
Fixed now. I've added scripts to package.json so we can easily run prettier manually.

e.g.

```
npm run prettier
```

```
npm run prettier-fix
```

We should also maybe take a look at [`lint-staged`](https://www.npmjs.com/package/lint-staged) to automate Prettier when committing changes locally?

Reply:
Your editor should pick up the prettier configuration from the repo; everything is there already since https://github.com/observablehq/plot/commit/87a6bcc2024dc37d5d55ee2ccd845c981a075e44 ; there is no need for additional tooling :) Also please keep concerns separate, this PR should only be about margin support.